db/migrate/20230920212922_create_bookmarks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBookmarks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookmarks do |t|
+      t.integer :id
```

Comment:
Got this error while running migrations in summary you should not define this id columns, those are by default created by rails

```
ou can't redefine the primary key column 'id'. To define a custom primary key, pass { id: false } to create_table.
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230920212922_create_bookmarks.rb:4:in `block in change'
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230920212922_create_bookmarks.rb:3:in `change'

Caused by:
ArgumentError: you can't redefine the primary key column 'id'. To define a custom primary key, pass { id: false } to create_table.
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230920212922_create_bookmarks.rb:4:in `block in change'
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230920212922_create_bookmarks.rb:3:in `change'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)

```